superbench/benchmarks/micro_benchmarks/sharding_matmul.py

```diff
@@ -221,7 +217,7 @@ def _benchmark(self):
         K = self._args.k
         N = self._args.n
         for mode in self._args.mode:
-            if mode == ShardingMode.NOSHARDING or self.__world_size == 1:
+            if mode == ShardingMode.NOSHARDING:
```

Comment:
Why remove self.__world_size == 1?

Reply:
The reason is introduced in the description of PR:

> For self.__matmul_nosharding() change the condition from 'if mode == ShardingMode.NOSHARDING or self.__world_size == 1:' to 'if mode == ShardingMode.NOSHARDING', since sharding-matmul with 1GPU should also be OK to run.